torch/nn/parameter.py

```diff
@@ -25,3 +25,8 @@ def __new__(cls, data=None, requires_grad=True):
 
     def __repr__(self):
         return 'Parameter containing:\n' + super(Parameter, self).__repr__()
+
+    def __reduce_ex__(self, proto):
```

Comment:
You're not using proto here. Should we use it?
If we don't need to use proto I think __reduce__ does the same thing but without the proto arg?